Source/WebCore/platform/graphics/cocoa/WebCoreDecompressionSession.mm

```diff
@@ -54,25 +54,30 @@
 
 namespace WebCore {
 
-static bool s_canCopyFormatDescriptionExtension = false;
-
-WebCoreDecompressionSession::WebCoreDecompressionSession(Mode mode)
-    : m_mode(mode)
-    , m_decompressionQueue(WorkQueue::create("WebCoreDecompressionSession Decompression Queue"_s))
+static bool canCopyFormatDescriptionExtension()
 {
+    static bool canCopyFormatDescriptionExtension = false;
 #if PLATFORM(VISION)
     static std::once_flag onceFlag;
     std::call_once(onceFlag, [] {
-        s_canCopyFormatDescriptionExtension = PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HasLeftStereoEyeView()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HasRightStereoEyeView()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HeroEye()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HorizontalDisparityAdjustment()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_StereoCameraBaseline()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_ProjectionKind()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_ViewPackingKind()
-        && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_CameraCalibrationDataLensCollection();
+        canCopyFormatDescriptionExtension = PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HasLeftStereoEyeView()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HasRightStereoEyeView()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HeroEye()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_HorizontalDisparityAdjustment()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_StereoCameraBaseline()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_ProjectionKind()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_ViewPackingKind()
+            && PAL::canLoad_CoreMedia_kCMFormatDescriptionExtension_CameraCalibrationDataLensCollection();
     });
 #endif
+    return canCopyFormatDescriptionExtension;
+}
+
+WebCoreDecompressionSession::WebCoreDecompressionSession(Mode mode, NSDictionary *pixelBufferAttributes)
+    : m_mode(mode)
```

Comment:
Maybe we do not need `m_mode` anymore, and we could just keep `pixelBufferAttributes`.